include/sensr_message_listener.h

```diff
@@ -23,7 +23,7 @@ namespace sensr {
     MessageListener(ListeningType listening_type = ListeningType::kOutputMessage);
     virtual ~MessageListener() = default;
     virtual void OnError(Error error, const std::string& reason);
-    virtual void OnGetOutpuMessage(const sensr_proto::OutputMessage &message);
+    virtual void OnGetOutputMessage(const sensr_proto::OutputMessage &message);
```

Comment:
Jerone mentioned to mark the old API as deprecated even it is miss-spelled.
I gonna handle this in the other PR.